src/e2e/puppeteer/__tests__/render-thoughts.ts

```diff
@@ -116,16 +138,14 @@ const testSuite = () => {
     })
 
     it('superscript', async () => {
```

Comment:
This may not be relevant after moving the light tests out of `testSuite`, but note that you can take advantage of nested describe blocks for clean organization of test variants:

## Option 1

```ts
	describe('initial load', () => {
	  it('dark', async () => {
	    ...
	  })
	  it('light', async () => {
	    ...
	  })
	})
```

## Option 2

```ts
	describe('dark', () => {
	  it('initial load', async () => {
	    ...
	  })
	})
	
	describe('light', () => {
	  it('initial load', async () => {
	    ...
	  })
	})
```